xpdacq/beamtime.py

```diff
@@ -77,6 +77,25 @@ def _configure_pe1c(exposure):
 
 
 def ct(dets, exposure, *, md=None):
+    """
+    Take one reading from area detectors with given exposure time
+
+    Parameters
+    ----------
+    detectors : list
+        list of 'readable' objects
+    exposure : float
+        total time of exposrue in seconds
+    md : dict, optional
+        extra metadata
```

Comment:
extra metadata what?  add more. Basically, if md is specified here, when will it be saved?  Every time ct is run, or just this instance?